tests/modules/rust-log/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "rust-log"
+version = "0.1.0"
+edition = "2021"
+authors = [ "Nitish Malhotra <nitishm@microsoft.com>" ]
+
+
+[lib]
+    crate-type = [ "cdylib" ]
+
+[dependencies]
+    wit-bindgen-rust = { git = "https://github.com/bytecodealliance/wit-bindgen", rev = "dab589f579b980c81b39c8348400b88a8c4d175a" }
```

Comment:
This should be revision `2e654dc82b7f9331719ba617a36ed5967b2aecb0`

Reply:
Will update. I think I picked it up before the update was merged.